examples/server/server.cpp

```diff
@@ -2986,6 +2986,7 @@ int main(int argc, char ** argv) {
                 }, [&](json error_data) {
                     server_sent_event(sink, "error", error_data);
                 });
+                sink.write("data: [DONE]\n\n", 14);
```

Comment:
this could be a static string, to prevent using magic number:

```suggestion
                static const std::string ev_done = "data: [DONE]\n\n";
                sink.write(ev_done.data(), ev_done.size());
```

Reply:
@ngxson 
suggest accepted.
and the failed tests are fixed.